.circleci/config.yml

```diff
@@ -81,7 +81,8 @@ jobs:
 
       - run: bazel run @nodejs//:yarn
       - run: bazel build ...
-      - run: bazel test ... --define=some_env=some_value
+      - run: bazel test ...
+      - run: bazel test //examples/define_var --define=some_env=some_value
```

Comment:
let's just remove this line, we already test this on the BazelCI, we should avoid duplicating things in circleCI as a step to removing it

Reply:
kk